scripts/tasks/jest.js

```diff
@@ -9,3 +9,11 @@ exports.jest = () =>
     ...(process.env.TRAVIS || argv().production ? { coverage: true } : undefined),
     ...(argv().u || argv().updateSnapshot ? { updateSnapshot: true } : undefined)
   });
+
+exports.jestWatch = () =>
+  jestTask({
+    ...(process.env.TRAVIS && { runInBand: true }),
+    ...(process.env.TRAVIS || argv().production ? { coverage: true } : undefined),
```

Comment:
just wondering why you just use `&&` in line 15 and then ternary in line 16 and 17..

Reply:
ah, no reason... perhaps to confuse and amaze.